lib/main.js

```diff
@@ -79,4 +83,22 @@ module.exports = {
 
 }
 
+function _findFileInParents(path, encoding, silent) {
+  for (var i = 0; i < 100; i++) {
+    try{
+      fs.readFileSync(path, { encoding: encoding })
+      return path
+    }
+    catch (e){
+      path = '../' + path
```

Comment:
biggest issue: paths need to be platform agnostic. you'll most likely want to use `path.resolve` but you'll need a starting point like `process.cwd()` which runs the risk of breaking people's current usage. needs lots of validation it works as expected